docs/src/loo_compare.md

```diff
@@ -34,52 +33,46 @@ data = rand(dist, 1000)
 
 The following code block defines the model along with its prior distributions using Turing.jl. We'll use this model with different fixed values for the `k` parameter.
 
-```@example loo_compare
-@model function model_LBA(data, k; min_rt=0.2)
+```@example loo_example
+@model function model_LBA(data, k; min_rt=minimum(data.rt))
     # Priors
     ν ~ MvNormal(fill(3.0, 2), I * 2)
     A ~ truncated(Normal(0.8, 0.4), 0.0, Inf)
     τ ~ Uniform(0.0, min_rt)
 
     # Likelihood
-    for i in 1:length(data)
-        data[i] ~ LBA(; ν, A, k, τ)
-    end
+    data ~ LBA(; ν, A, k, τ)
 end
 ```
 
 ## Estimate the Parameters
 
 Now we'll estimate the parameters using three different fixed `k` values:
 
-```@example loo_compare
-#prepare the data for model fitting
-dat = [(choice=data.choice[i], rt=data.rt[i]) for i in 1:length(data.rt)]
-min_rt = minimum(data.rt)
-
-chain_LBA1 = sample(model_LBA(dat, 2,  min_rt=min_rt), NUTS(), 1000)
-chain_LBA2 = sample(model_LBA(dat, 0.3,min_rt=min_rt), NUTS(), 1000)
-chain_LBA3 = sample(model_LBA(dat, 1,  min_rt=min_rt), NUTS(), 1000)
+```@example loo_example
+chain_LBA1 = sample(model_LBA(data, 2.0), NUTS(), 1000)
+chain_LBA2 = sample(model_LBA(data, 0.3), NUTS(), 1000)
+chain_LBA3 = sample(model_LBA(data, 1.0), NUTS(), 1000)
 ```
 
 ## Compute PSIS-LOO
 
 Next we will use the `psis_loo` function to compute the PSIS-LOO for each model:
 
-```@example loo_compare
-res1 = psis_loo(model_LBA(dat, 2,   min_rt=min_rt), chain_LBA1)
-res2 = psis_loo(model_LBA(dat, 0.3, min_rt=min_rt), chain_LBA2)
-res3 = psis_loo(model_LBA(dat, 1, min_rt=min_rt), chain_LBA3)
-show(stdout, MIME"text/plain"(), ans) #hide
+```@example loo_example
+res1 = psis_loo(model_LBA(data, 2.0), chain_LBA1)
+res2 = psis_loo(model_LBA(data, 0.3), chain_LBA2)
+res3 = psis_loo(model_LBA(data, 1.0), chain_LBA3)
+show(stdout, MIME"text/plain"(), ans)
```

Comment:
Documenter.jl will only hide the fix to the output of the pretty_table that psis_loo outputs with the comment present.